joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/SingleAssignmentTests.scala

```diff
@@ -289,4 +289,84 @@ class SingleAssignmentTests extends RubyCode2CpgFixture {
       case xs => fail(s"Expected three lambdas, got ${xs.size} lambdas instead")
     }
   }
+
+  "Bracketed ||=" in {
+    val cpg = code("""
+        |hash[:id] ||= s[:id]
+        |""".stripMargin)
+
+    inside(cpg.call.name(Operators.assignmentOr).l) {
```

Comment:
I believe this is conditional assignment, not assign or https://stackoverflow.com/a/14697343
Probably the case that we need to lower this into:
```ruby
if hash[:id].nil? then
  hash[:id] = s[:id]
end
```